src/util/handlebars/handlebar-helpers.ts

```diff
@@ -27,6 +27,9 @@ export const registerHandlebarsHelpers = () => {
 				: `${numberOfSyncedRepos} / ${totalNumberOfRepos}`
 	);
 
+	//check to display repo count as zero in case of org with zero repos
+	hbs.registerHelper("hasTotalNumberOfRepos", (totalNumberOfRepos: number) => (totalNumberOfRepos || totalNumberOfRepos === 0));
```

Comment:
This helper name is a bit confusing, can we change it to something like `checkRepos` or something.
Also, I think its better to do a check to `totalNumberOfRepos >= 0` than `OR` check cause this `OR` check will fail for unlikely scenarios like negative numbers.